### ISSUE TITLE
Mobile: remove link either from the social media icon or the text

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#873]
Android 12/Talk back/ Chrome

For individuals and organizations, both the social media icon and the text are linked. Screen reader reads the same link twice. It's better to only link the icon or the text and not both.

**Comment by cherylhjli** [comment]
Let me know if you need a design mockup or if you're good to make the changes without a mockup.

**Comment by greatislander** [comment]
This should be resolved now. I've added `aria-hidden="true"` to the icon which should prevent it from being announced.
